README.md

```diff
@@ -10,27 +10,27 @@ software.
 Rather than operating as a single, monolithic entity, KubOS is comprised of a series
 of independent, yet interoperating components.
 
-<p align="center"> 
+<p align="center">
 <img src="docs/images/architecture_stack.png">
 </p>
 
 - Mission applications control and execute the logic necessary to accomplish mission goals
 - Services expose hardware and system functionality with a controlled and uniform interface
 - Kubos Linux provides the base operating system and the drivers needed to communicate with connected hardware devices
 
-[Company website](https://www.kubos.com)
+[Company website](https://kubos-preservation-group.github.io/)
```

Comment:
small tweak, but could you update this to say "community website" rather than "company website"?